packages/sui-react-web-vitals/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@s-ui/react-web-vitals",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "lib/index.js",
+  "scripts": {
+    "lib": "babel --presets sui ./src --out-dir ./lib",
+    "prepublishOnly": "npm run lib",
+    "test:browser:watch": "NODE_ENV=test npm run test:browser -- --watch",
+    "test:browser": "NODE_ENV=test sui-test browser",
+    "test:server:watch": "npm run test:server -- --watch",
+    "test:server": "NODE_ENV=test sui-test server",
+    "test": "npm run test:server && npm run test:browser"
```

Comment:
Is this used?

Reply:
No but I will leave it for now because I'm planning to add tests